python/cudf/cudf/_libxx/aggregation.pxd

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) 2020, NVIDIA CORPORATION.
 
 from libcpp.string cimport string
+from libcpp.memory cimport unique_ptr
```

Comment:
How come this was added?

Reply:
Question applies to multiple files.

Reply:
We're using `unique_ptr` here but it's not explicity `cimport`'d.